### DISCUSSION TITLE
'Unexpected channel configuration change"

### DISCUSSION THREAD

**Discussion by bitwombat** [audacious-media-player/audacious#1390] [Q&A] [answered]
Using Audacious 4.3.1 (and earlier), all of the ABC-owned streaming channels here in Australia play for a second then there's a pink error at the bottom of Audacious that says: 'Unexpected channel configuration change".

Example stream: http://live-radio01.mediahubaustralia.com/2TJW/aac/

Rhythmbox is OK with them.

I see this in stdout:

```
ERROR aac.cc:441 [play]: Bitstream value not allowed by specification.
ERROR aac.cc:441 [play]: Unexpected channel configuration change.
ERROR aac.cc:441 [play]: Unexpected channel configuration change.
ERROR aac.cc:441 [play]: Unexpected channel configuration change.
ERROR aac.cc:441 [play]: Channel coupling not yet implemented.
ERROR aac.cc:441 [play]: Unexpected channel configuration change.
ERROR aac.cc:441 [play]: Unexpected channel configuration change.
ERROR aac.cc:441 [play]: Unexpected channel configuration change.
```

Is this a feature of the stream that Audacious should be OK with (ie. a bug), or something not yet implemented (as hinted above), or am I pointing at the wrong URL?

**Comment by radioactiveman** [comment, marked as answer]
Audacious uses [faad2](https://github.com/knik0/faad2) for the AAC plugin. The error you see is coming from this library.

I can reproduce it with Ubuntu 20.04 (with libfaad2 2.9.1) but not on Arch Linux with the current faad2 version 2.11.1. The stream plays fine here without any error message.

What is your distribution and faad2 version?

**Reply by bitwombat** [reply]
Ah, thanks @radioactiveman. Yeah, using Ubuntu 20.04 with libfaad2 2.9.1. I'll find a faad2 2.11.1 .deb file and `dpkg -i` it to patch my system.

**Reply by bitwombat** [reply]
Can confirm [libfaad2_2.10.1-1](https://packages.debian.org/bookworm/libfaad2) also works. Needed the older one because of the required libc6 version.
